chaoslib/litmus/cpu_hog/pod_cpu_hog.yml

```diff
@@ -92,14 +88,14 @@
       ## considering duration in milliseconds from spec
       ## with a grace period of 2 min before cpu stress job terminate
       set_fact:
-        job_wait_time: "{{ ((c_duration|int)/1000 + 120)| int }}"
+        job_wait_time: "{{ ((c_duration|int) + 120)| int }}"
```

Comment:
with 120 we don't need int 
it can be `"{{ ((c_duration | int) + 120) }}"`

Reply:
I think this command is fine. If we typecast the whole expression once. It would ensure that the whole expression is in int ( double-check). It won't be a problem.

Reply:
That is fine, but why do we need an extra check on this. This is a refactor PR best one to correct these changes.
@ksatchit

Reply:
FYI, Typecast the whole expression once is the best one for these changes.